apps/chrome-devtools/src/app-devtools/app.component.ts

```diff
@@ -1,21 +1,33 @@
-import { AsyncPipe } from '@angular/common';
-import { ChangeDetectionStrategy, Component, OnDestroy } from '@angular/core';
+import { AsyncPipe, JsonPipe } from '@angular/common';
+import { ChangeDetectionStrategy, Component, computed, effect, inject } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { FormBuilder, FormControl, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { DfSelectModule, DfTooltipModule } from '@design-factory/design-factory';
 import { NgbNavModule } from '@ng-bootstrap/ng-bootstrap';
-import { type RulesetExecutionDebug, RulesetHistoryPresModule } from '@o3r/rules-engine';
-import { Observable, Subscription } from 'rxjs';
+import { RulesetHistoryPresModule } from '@o3r/rules-engine';
 import { AppConnectionComponent } from '../components/app-connection/app-connection.component';
-import { ComponentPanelPresComponent } from './component-panel/component-panel-pres.component';
+import type { State } from '../extension/interface';
+import { StateService } from '../services';
 import { ChromeExtensionConnectionService, isApplicationInformationMessage } from '../services/connection.service';
 import { RulesetHistoryService } from '../services/ruleset-history.service';
+import { ComponentPanelPresComponent } from './component-panel/component-panel-pres.component';
 import { ConfigPanelPresComponent } from './config-panel/config-panel-pres.component';
 import { DebugPanelPresComponent } from './debug-panel/debug-panel-pres.component';
 import { DebugPanelService } from './debug-panel/debug-panel.service';
 import { LocalizationPanelPresComponent } from './localization-panel/localization-panel-pres.component';
+import { StatePanelComponent } from './state-panel/state-panel.component';
 import { ThemingPanelPresComponent } from './theming-panel/theming-panel-pres.component';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
+  styles: `
+    ::ng-deep ng-select.local-change .ng-select-container {
```

Comment:
can't we put this in the global scss file ?
if not, `::ng-deep` should probably be `:host ::ng-deep` instead to break a bit less view encapsulation

Reply:
There is no style css file, so I guess not for now :D